src/main/java/org/sagebionetworks/web/client/presenter/SearchPresenter.java

```diff
@@ -89,6 +91,15 @@ public void setPlace(Search place) {
     String queryTerm = place.getSearchTerm();
     if (queryTerm == null) queryTerm = "";
     currentSearch = checkForJson(queryTerm);
+    if (currentSearch.getQueryTerm().size() == 1) {
```

Comment:
If there's a single search term, check to see if we should redirect (to Synapse entity page or People Search)